README.md

```diff
@@ -159,23 +161,28 @@ includes:
 - Jaeger Trace Exporter
 - Trace Sampling
 
+The [v0.3 alpha
+release](https://github.com/open-telemetry/opentelemetry-python/releases/tag/v0.3.0)
+includes:
+
+- Metrics Instruments and Labels
+- Flask Integration
+- PyMongo Integration
+
 See the [project
 milestones](https://github.com/open-telemetry/opentelemetry-python/milestones)
 for details on upcoming releases. The dates and features described here are
 estimates, and subject to change.
 
 Future releases targets include:
 
-| Component                           | Version    | Target Date     |
-| ----------------------------------- | ---------- | --------------- |
-| Zipkin Trace Exporter               | Alpha v0.3 | December 6 2019 |
-| W3C Correlation Context Propagation | Alpha v0.3 | December 6 2019 |
-| Support for Tags/Baggage            | Alpha v0.3 | December 6 2019 |
-| Metrics Aggregation                 | Alpha v0.3 | December 6 2019 |
-| gRPC Integrations                   | Alpha v0.3 | December 6 2019 |
-| Prometheus Metrics Exporter         | Alpha v0.3 | December 6 2019 |
-
-| Component              | Version    | Target Date      |
-| ---------------------- | ---------- | ---------------- |
-| OpenCensus Bridge      | Alpha v0.4 | December 31 2019 |
-| Metrics SDK (Complete) | Alpha v0.4 | December 31 2019 |
+| Component                           | Version    | Target Date      |
+| ----------------------------------- | ---------- | ---------------- |
+| Zipkin Trace Exporter               | Alpha v0.4 | December 31 2019 |
```

Comment:
will we release v0.4 on new years? seem like it's worth pushing by a week.

Reply:
Yeah, this date is a placeholder. Let's come up with a real date on the SIG call.